packages/web-components/src/components/va-additional-info/va-additional-info.tsx

```diff
@@ -99,8 +104,10 @@ export class VaAdditionalInfo {
             <i class="fa-angle-down" role="presentation" />
           </span>
         </a>
-
-        <div id="info" class={this.open ? 'open' : 'closed'}>
+        <div
+          id={this.disableBorder ? 'info-no-border' : 'info'}
```

Comment:
What does changing the `id` do for us?

Reply:
@bkjohnson it makes it so the extra CSS that creates the padding left and blue border doesn't get added in: https://github.com/department-of-veterans-affairs/component-library/blob/f13301ccda884d93e8fcd68ee36c24bf4a8b1c70/packages/web-components/src/components/va-additional-info/va-additional-info.css#L5-L9

Reply:
I feel like changing the `id` isn't super obvious. Are we able to accomplish this with an attribute selector instead?

Reply:
@bkjohnson I have reworked to use the :host([disable-border='false']) selector to differentiate when the css should be added in